wuvt/defaults.py

```diff
@@ -30,7 +30,7 @@
 
 # DJ activity timer for automation/DJ logout in seconds
 DJ_TIMEOUT = 30 * 60
-EXTENDED_DJ_TIMEOUT = 120 * 60
+EXTENDED_DJ_TIMEOUT = 240 * 60
```

Comment:
any particular reason for this? no DJ should be playing tracks longer than 2 hours reasonably. I played bach's mass in B minor and cheated on the logging, and it was still only about 2 hours of music.